src/plugin/electron/electronconfig.js

```diff
@@ -0,0 +1,57 @@
+goog.provide('plugin.electron.ElectronConfigCtrl');
+goog.provide('plugin.electron.electronConfigDirective');
+
+goog.require('os.ui.Module');
+
+/**
+ * The electron configuration directive.
+ * @return {angular.Directive}
+ */
+plugin.electron.electronConfigDirective = function() {
```

Comment:
We might want to be more consistent with the directive/template/file naming. Is this intended to be general Electron configuration or just memory config?

Reply:
changed the naming to be more consistent with memory config.